source/server-discovery-and-monitoring/tests/unified/serverMonitoringMode.yml

```diff
@@ -0,0 +1,67 @@
+description: serverMonitoringMode
+
+schemaVersion: "1.3"
+
+tests:
```

Comment:
I was expecting these tests to assert on the value of `ServerHeartbeatSucceedEvent.awaited`.  As it is I'm not clear what use these tests are.  Can you explain?

Reply:
Good call out. I've updated the unified test format to support capturing these SDAM events and added the awaited=True/False assertions to these tests.